src/PointsOfInterest/ResponseRenderer.php

```diff
@@ -74,6 +74,8 @@ public function getResponse(Template $template, Model $model, Request $request):
                 $tempPoi['icon'] = $poiIcon;
             }
 
+            $tempPoi['label'] = $poi['addCustomLabel'] ? $poi['customLabel'] : 'More';
```

Comment:
I would leave it `null` when not custom, so users can easily adjust it in the template.